lib/elixir_internal_certificate_web/router.ex

```diff
@@ -82,9 +82,9 @@ defmodule ElixirInternalCertificateWeb.Router do
   scope "/", ElixirInternalCertificateWeb do
     pipe_through [:browser, :require_authenticated_user]
 
-    get "/", UserSearchController, :index
+    resources "/keywords", UserSearchController, only: [:index, :show]
 
-    get "/keywords/:id", UserSearchController, :show
+    get "/", PageController, :index
```

Comment:
not sure the purpose of that controller so far :D

Reply:
I want to redirect all request from "{host_url}" to "{host_url}/keywords" @andyduong1920